pkg/services/resourceFilterService_test.go

```diff
@@ -203,6 +203,29 @@ func TestWhiteListFilters(t *testing.T) {
 		assert.Equal(t, true, ShouldResourceBeValidated(admissionReviewReq, rootObject))
 	})
 
+	t.Run("resource should be validated because it has a system: username and annotations openshift.io/requester and the value is not prefix system:serviceaccount", func(t *testing.T) {
+		admissionReviewReq, rootObject := extractAdmissionReviewReqAndRootObject(templateResource)
+		admissionReviewReq.Request.UserInfo.Username = "system:test-test"
+		rootObject.Metadata.ManagedFields[0].Manager = "openshift"
```

Comment:
i think its necessary because we get the `true` from `isKubectl`

![image](https://github.com/datreeio/admission-webhook-datree/assets/29673827/03674ace-cd69-4e27-b55e-3d5363f55e7d)